.github/workflows/ci-build.yml

```diff
@@ -0,0 +1,49 @@
+name: CI Build
+
+on:
+  - push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        terraform_version: [0.12.x, 0.13.x, 0.14.x, 0.15.x, 1.0.11, latest]
+
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Prepare bin directory
+        run: |
+          mkdir -p ${HOME}/bin
+          echo '${HOME}/bin/' >> $GITHUB_PATH
+
+      - name: Install shellcheck
+        run: |
+          wget "https://github.com/koalaman/shellcheck/releases/download/stable/shellcheck-stable.linux.x86_64.tar.xz"
+          tar -xvf shellcheck-stable.linux.x86_64.tar.xz
+          cp shellcheck-stable/shellcheck ${HOME}/bin/
+          rm -rf shellcheck-stable*
+          shellcheck --version
+
+      - name: Install terraform
+        uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_version: ${{ matrix.terraform_version }}
```

Comment:
I think we wanted to remove GitHub actions and replace them by bash scripts...
Maybe you can check how much effort it would be?

Reply:
Yes, I also wanted them to be in the bash. Couple commits before they were in bash, but I could not get `tflint` installation work. I switched to action for terraform install since it is managed also by Hashicorp.

Let me look into it, maybe we can keep them in bash.